src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -198,7 +189,9 @@ protected void onCreate(Bundle savedInstanceState) {
 
         mRedirectUrl = mAuthRequest.getRedirectUri();
         Logger.v(TAG, "OnCreate redirectUrl:" + mRedirectUrl);
-        setupWebView();
+     // Create the Web View to show the page
+        mWebView = (WebView)findViewById(this.getResources().getIdentifier("webView1", "id",
```

Comment:
It is inside the existing layout file. It will be breaking change for developers using this layout.